metrics/README.md

```diff
@@ -19,3 +19,21 @@ Once you have these things, run
 
 which should generate new metrics and place them into `fontMetrics.js`. You're
 done!
+
+### OS X Notes
+
+Install the JSON perl module:
+    
+    sudo cpan JSON
+
+Install fontforge with python extensions (this may install python as well):
+
+    brew install fontforge --enable-pyextension
+    
+If you get a "Segmentation fault: 11" when running `make metrics` you'll have
```

Comment:
The shebang has to contain a full path, the `PATH` environment variable doesn't have any effect. One common solution is writing it as

    #!/usr/bin/env python2

hoping that that's the name which is being used, and that it's on the path.